### DISCUSSION TITLE
Commits disappeared (org.codespaces_ownership_updated)

### DISCUSSION THREAD

**Discussion by Dam2432** [community/community#75125] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I recently lost my work after following a `org.codespaces_ownership_updated` audit log event on github.
When it happened, all commits from the last 3 months disappeared. I did not perform any action on github by myself, and I don't really understand what this log means. 
Ideally, I would like to understand what happened and get my work back. 
Do I have to update my plan if it is related to codespaces? I'm not really confortable on github and don't want to make an irreparable mistake. 

Here are the details of the event :
**[org.codespaces_ownership_updated]**
**@timestamp**	2023-10-19 22:15:11 +0200
**_document_id**	ks659wD7BR0XVRfmmqW4JQ
**action**	org.codespaces_ownership_updated
**actor**	DevMyOwnGames
**actor_id**	125405271
**created_at**	2023-10-19 22:15:11 +0200
**operation_type**	modify
**org**	DevMyOwnGames
**org_id**	125405271
**owner_type**	User
**user_agent**	Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 

Thank you for reading.

**Comment by davixmns** [comment]
Here are some steps you can take to investigate and potentially recover your lost work:

Review Codespaces Ownership:
Check the ownership and permissions of the Codespaces in your organization. Ensure that your user account (DevMyOwnGames) has the necessary ownership and access rights.

Contact GitHub Support:
Reach out to GitHub Support to report the issue and seek assistance. They can investigate the event and provide guidance on potential recovery options. You can explain the situation and provide the details of the log event you shared here.

Backup and Repository Cloning:
If you have a local clone of your repository or any backup, consider checking if the lost commits are still available in your local copy. If so, you can push them back to the GitHub repository.

Check Repository Settings:
Review the settings of your GitHub repository, especially the branch settings. Ensure that there are no unusual changes or restrictions that might have affected the visibility of commits.

Audit Log:
Continuously monitor the GitHub audit log for any further events related to Codespaces or ownership changes. This might provide additional information about what happened.

Update Plan (if necessary):
Check if there are any limitations or changes in your GitHub plan that might have affected Codespaces usage. If an update is required, GitHub will typically notify you about any plan-related issues.

Exercise Caution:
When dealing with sensitive operations, it's always a good idea to exercise caution. If you are unsure about making any changes, it's better to seek guidance from GitHub support or community forums before proceeding.
